Sources/BraintreeCore/BTSessionManager.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+/// :nodoc: This method is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+///
+/// Manages a shared session ID across all Braintree payment components to enable end-to-end analytics tracking.
+
+@_documentation(visibility: private)
+public class BTSessionManager {
```

Comment:
If we don't expect to add anything to this in the future we can rename to:

```suggestion
public class BTSessionIDManager {
```